### ISSUE TITLE
(Description) Check screen reader behaviour

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#243]
Before RC we need to do sanity checks on screen reader experience for any major issues.

- [ ] VoiceOver
- [ ] iOS VoiceOver
- [ ] NVDA
- [ ] JAWS

**Note** that in BASE issue https://github.com/phetsims/balloons-and-static-electricity/issues/508 we deferred some work in common code that would fix the mobile drag and grab interaction. Currently the visual keyboard grab hint shows up on touch devices. 

We may need to continue to defer mobile accessibility for Friction as well if this common code issue cannot be addressed in a timely fashion.

**Comment by terracoda** [comment]
**VoiceOver issues on MacOs 11.4**
1. The reset hint has been permanently appended to the settled state, "Atoms jiggle a tiny bit, temperature cool. Reset sim to make more observations."
    - the reset hint has to be conditionally appended. It can't be heard of atoms are available to be evaporated.

2. There is a huge amount of pile-ups. Hopefully, we have modernized the management of responses and it will be straightforward to address the pileups.

**Comment by terracoda** [comment]
**VoiceOver issues on iOS 11.4**
1. See visual keyboard hint on touch device (iphone) as noted in https://github.com/phetsims/balloons-and-static-electricity/issues/508
  - this needs to fixed in common code, or we leave iOS off for this release

2. Hear ticking as sim loads (this must be built-in apple sonification)
3.  Can't access sim loaded from phettest @zepumph, please provide latest dev link here. I was using the sim yesterday or the day before with iOS VoiceOver.

**Comment by terracoda** [comment]
VoiceOver issues on iOS 14.6 (iPhone 8)
1. keyboard grab hint visible (same as above)
2. significant response pile ups happen
3. hints like move down to rub harder and rub fast or slow seem to interrupt more important warming-cooling responses and break away responses

**Comment by terracoda** [comment]
@Matthew-Moore240 please do a quick test with NVDA to see if there are obvious issues with pileups on that platform as well.

**Comment by zepumph** [comment]
From testing on NVDA:

* In general it sounded good.
* Easy to hear the break-away alert interrupted, see https://github.com/phetsims/friction/issues/242#issuecomment-931747740
* I also felt like there could be too many "move down to rub harder" alerts, so I created https://github.com/phetsims/friction/issues/252.

**Comment by zepumph** [comment]
On hold until https://github.com/phetsims/friction/issues/253 can be fixed. That issue is too large to get good results from other testing.

**Comment by terracoda** [comment]
Also noting this version of Friction will not be published with iOS Interactive Description, so we will not be addressing https://github.com/phetsims/balloons-and-static-electricity/issues/508

**Comment by zepumph** [comment]
Agreed! See https://github.com/phetsims/friction/issues/249 for notes on the discussion we had.

**Comment by terracoda** [comment]
I checked VO on MacOS and VO on iOS. Both sound pretty good. VO on iOS will need a bit more work, but this version is being published with iOS A11y.

For iOS, just noting along with the visual keyboard hint, I had some pile ups of move down to rub harder.

**Comment by Matthew-Moore240** [comment]
This all sounded really good to me. The only thing was that I had to do the long swipes across the book to get the now hotter and warming alerts but I had no pile-up or anything like that.

**Comment by zepumph** [comment]
Ready to close. Thanks!
